### ISSUE TITLE
Having small ? inside black circles for hovertips?

### ISSUE THREAD

**Issue by paaljoachim** [sybrew/the-seo-framework#215]
I am now on the SEO Settings - General Settings - Layout.

Looking at SEO Bar Settings. Wondering if I should click the Display the SEO Bar in the SEO Settings metabox. Through out the settings there could be small black circles with ? in them. Hovering over these can bring up a tool tip help text. OR just hovering over the text could bring up a tooltip. So lets say that I hovered over "Display the SEO Bar in the SEO Settings metabox" then at tooltip could show up giving extra info.

Hmm I am hesitant with the above suggestion. As it might not look that good visually.

**Comment by sybrew** [comment]
I've been playing with such elements in extensions:

![image](https://user-images.githubusercontent.com/7662236/32945021-25a3c11a-cb92-11e7-990c-4419dd17a057.png)

I do think all those circles will be overwhelming (i.e. seen as bloat) by some users.

That said, the tooltips you speak of are being shipped with 3.0 😄

**Comment by LeBaux** [comment]
Personally, I don’t think there is such a thing as “too much help” when it comes to SEO. TSF caters to various users with widely different levels of expertise. One clear downside is that the sheer amount of tooltips piles up new strings for translators, potentially hindering the initial adaptation of TSF among non–English speakers.

**Comment by sybrew** [comment]
@LeBaux From TSF 3.0 only 559 strings are translatable ☺️

**Comment by paaljoachim** [comment]
Good to hear that tooltips will be included in 3.0!..:)

**Comment by sybrew** [comment]
I've played with the Dashicons as a replacement, and they'll be very overwhelming:
1. There will be many of them.
2. WordPress' flat and generic layout doesn't play nicely with inline bold icons.

We might ever replace them with intuitive links, like within explanatory sentences.
Alternatively, we need to update the layout to what the Extension Manager is using, which isn't generic. Now, the rise of Gutenberg's new interface will be another great starting point.

I'm closing this indefinitely as the proposed implementation doesn't fit within the current layout. We'll see where the road will lead us when we change the complete layout.

**Comment by paaljoachim** [comment]
Hey Sybre. Put me on your beta/alpha testing list as I would like to help out. I do have feedback in regards to the extension page as it just feels to tight. I will look into it and create an issue for it.
